Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:2.5.3
+
+RUN apt-get update -qq && apt-get install -y nodejs postgresql-client
```

Comment:
Good job Gloria.
I see you are installing `nodejs`.I don't think we need `nodejs dependencies` since this is an api only